CHANGELOG.rst

```diff
@@ -25,6 +25,9 @@ Fixed
 - ``add_subcommands`` fails when parser has required argument and default config
   available `#232 <https://github.com/omni-us/jsonargparse/issues/232>`__.
 
+- ``fail_untyped`` does not work with dataclasses
```

Comment:
Also, please move this to the end of the `Added` section without leaving any space in between. And change it to something like ` ``add_dataclass_arguments`` now supports the ``fail_untyped`` parameter `.

Reply:
Right, done!

Reply:
What I meant was to remove it from Fixed since it is not a bug fix. It is a new feature. Remove this item and move the link to the Added section.

Reply:
sure, fixed!